goopicsi.go

```diff
@@ -279,13 +284,13 @@ func ExposeRemoteNVMe(subsystemID string, subsystemNQN string, maxNamespaces int
 		})
 		if err != nil {
 			log.Println(err)
-			return err
+			return "", err
 		}
 		log.Printf("NVMe Controller created: %v", response2)
-		return nil
+		return subsystemID, nil
 	}
 	log.Printf("NVMe Controller is already present with the controllerID: %v", controllerID)
-	return nil
+	return subsystemID, nil
```

Comment:
you also need to return `controllerID`
otherwise, the value will be lost and you can't re-use it
for example, when you want to remove the controller...

Reply:
@glimchb made the requested changes.

Reply:
@glimchb Done